lib/experimental/posts/load.php

```diff
@@ -7,22 +7,95 @@
 
 add_action( 'admin_menu', 'gutenberg_replace_posts_dashboard' );
 
+// Default to is-fullscreen-mode to avoid jumps in the UI.
+add_filter(
+	'admin_body_class',
+	static function ( $classes ) {
```

Comment:
Normally anonymous functions are not recommended in core, because they don't allow a third-party plugin to use remove_filter.

Reply:
That's what we do for site editor in core right now. I'll have to check if I can add the filter only in this new page though.. I didn't observe something weird so far though..

Reply:
Maybe the current one for site editor, passed during a code review but as far as I know anonymous functions should be avoided, they work well the issue is if a plugin wants to remove the filter the plugin can not do it.